source/assets/javascripts/application.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function() {
+  $(".menu-trigger").on("click", showMobileMenu);
+
+  function showMobileMenu(e) {
+    e.preventDefault();
+
+    $(".nav-links").toggleClass("active");
+    $(".nav-links.active a").on("click", dismissMobileMenu);
+  }
+
+  function dismissMobileMenu(e) {
+    $(".nav-links").removeClass("active");
+  }
+});
```

Comment:
Would it be possible to extract this to something like `javascripts/mobile_menu.js`?